scripts/run_prover_profile.sh

```diff
@@ -0,0 +1,11 @@
+# This command will fetch the node statistics in memory, run the prover profile, and output the differences after the last task completes to the specified file.
+
+node dist/index.js prover-profile \
+    -r "service-endpoint" \
+    -u "user-address" \
+    -x "private-key" \
+    --num_prove_tasks 3 \
```

Comment:
quick question, did I miss something?
I only see the prover-profile do the node static get and compare and generation, didn't see it add prove tasks? This option is not used?